pkg/minikube/service/service_test.go

```diff
@@ -824,6 +845,7 @@ func TestWaitAndMaybeOpenService(t *testing.T) {
 		urlMode     bool
 		https       bool
 		err         bool
+		nondefault  bool
```

Comment:
bools should never be negative or have a name that starts with with `no`: it gets too confusing, particularly for non-english readers, when `nondefault=false`. I also don't understand what this bool is the default of.

Instead of `default` or `nondefault`, I think this would be more readable if each test directly defined which `servicesMap` to use.

Reply:
Will look into the test again and refactor to make it more readable for code structure

Reply:
Refactor the code to split out the test on it's own to make it more easier to understand for other. Let me know what you think ?. Thanks